website/client/package.json

```diff
@@ -52,5 +54,13 @@
     "vuedraggable": "^2.23.1",
     "vuejs-datepicker": "git://github.com/habitrpg/vuejs-datepicker.git#5d237615463a84a23dd6f3f77c6ab577d68593ec",
     "webpack": "^4.41.2"
+  },
+  "devDependencies": {
```

Comment:
can you move these deps to normal ones instead of devdeps? Otherwise it's not possible to build storybook when deploying because they're not installed in prod

Reply:
moved to normal dependencies